base/asyncevent.jl

```diff
@@ -206,6 +207,12 @@ end
 
 isopen(t::Union{Timer, AsyncCondition}) = @atomic :acquire t.isopen
 
+"""
+    close(t::Union{Timer, AsyncCondition})
+Close an object `t` and thus mark it as inactive. Once a timer or condition is inactive, it will not produce a new event.
+See also: [`isopen`](@ref)
+"""
+
 function close(t::Union{Timer, AsyncCondition})
```

Comment:
The blank line between the string and the method definition means that the string is not being attached to the method as a docstring